src/client/logic/slider/reducer.js

```diff
@@ -0,0 +1,16 @@
+import defaultState from 'client/logic/defaultState';
+import { SLIDE_CHANGE } from './actionTypes';
+
+export default function slideReducer(state = defaultState.slider, action) {
+    switch(action.type) {
+        case SLIDE_CHANGE: {
+            return {
+                ...state,
+                slideId: action.payload
```

Comment:
What is payload here, sliderId? So, your action will just change sliderId? What for?